packages/terra-action-footer/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Action Footer
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-action-footer.svg)](https://www.npmjs.org/package/terra-action-footer)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The terra-action-footer component is a footer bar that contains sockets to place items in the form of buttons and hyperlinks used to perform actions. The default variant contains a start and end socket, while the centered variant has only a center socket. If no actions are provided, the footer bar defaults to a 10px equivalent height and maintains the top border.
```

Comment:
Would a consumer ever want the default height to _not_ be 10px?

Reply:
Spoke with UX, we're gonna go ahead and make it themeable, but default it to 10px.